admin-dev/themes/default/scss/font.scss

```diff
@@ -1,35 +1,102 @@
-// stylelint-disable
-@font-face {
-  font-family: "Material Icons";
-  font-style: normal;
-  font-weight: 400;
-  src: local("Material Icons"), local("MaterialIcons-Regular"),
-    url("~material-design-icons-iconfont/dist/fonts/MaterialIcons-Regular.woff2")
-      format("woff2");
+/* stylelint-disable */
+$FontPathIBMPlexSans: "~@fontsource/ibm-plex-sans";
+$local-font: true;
+
+@mixin setFont() {
```

Comment:
Is it possible to use the same principle you changed recently as in `_import.scss` in the UI kit

Or maybe import this file from UI kit directly? Or maybe it's not even needed anymore to do this here since the fonts are already included in the UI kit?

Reply:
Yes, I have refactored it the same way 👍